tests/MockObjectTest.php

```diff
@@ -838,7 +838,8 @@ private function resetMockObjects()
     public function testStringableClassDoesNotThrow()
     {
         $mock = $this->getMock('StringableClass');
+        $mock->method('__toString')->willReturn('foo');
```

Comment:
you should write another test instead of modifying this one. Otherwise, the case of mocking __toString without defining a return value is not tested anymore